app/src/main/java/com/example/sharewhatyoucanproject/HomescreenActivity.kt

```diff
@@ -14,13 +14,17 @@ class HomescreenActivity : AppCompatActivity() {
         // donor action
         val button = findViewById<View>(R.id.donerimg) as ImageView
         button.setOnClickListener { // Do something in response to button click
-            startActivity(Intent(this@HomescreenActivity, LoginActivity::class.java))
+            val i = Intent(this@HomescreenActivity, LoginActivity::class.java)
+            i.putExtra("type", 1)
```

Comment:
use `UserType.DONOR.ordinal` so that the code is more readable

Reply:
-Done